app/views/login.erb

```diff
@@ -0,0 +1,7 @@
+<div class="content">
```

Comment:
The formatting and indentation of this file is wrong which makes it very hard to read.